src/cms/collections/createTasks.js

```diff
@@ -0,0 +1,82 @@
+const tasksRecord = (required = true) => [
+  {
+    name: 'id',
+    label: 'ID',
+    widget: 'string',
+    required,
+  },
+  {
+    name: 'points',
+    label: 'Points',
+    widget: 'number',
+    required,
+  },
+  {
+    name: 'icon',
+    label: 'Icon',
+    widget: 'string',
+    required,
+  },
+  {
+    name: 'title',
+    label: 'Title',
+    widget: 'string',
+    required,
+  },
+  {
+    name: 'description',
+    label: 'Description',
+    widget: 'string',
+    required,
+  },
+  {
+    name: 'questions',
+    label: 'Questions',
+    widget: 'list',
+    fields: [
+      {
+        name: 'title',
+        label: 'Title',
+        widget: 'string',
+        required,
+      },
+      {
+        name: 'description',
+        label: 'Description',
+        widget: 'string',
+        required: false,
+      },
+      {
+        name: 'format',
+        label: 'Format',
+        widget: 'string',
+        required,
+      },
+      {
+        name: 'option',
+        label: 'Option',
+        widget: 'string',
```

Comment:
Please make widget list as per https://www.netlifycms.org/docs/widgets/#list. default should be `[]`